setup.py

```diff
@@ -34,15 +34,16 @@ def load_about():
     },
     license="AGPLv3",
     author="Sofiane Bebert",
-    description="enterprisecatalog plugin for Tutor",
+    description="enterprise-catalog plugin for Tutor",
     long_description=load_readme(),
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
-    python_requires=">=3.5",
-    install_requires=["tutor"],
+    install_requires=["tutor>=16.0.0,<17.0.0"],
```

Comment:
Won't it work with tutor 17? Why do we want to enforce the upper version?

Reply:
We don't, but I have tested it on palm only. Generally all plugins enforce a specific version target.